src/api/app/views/webui/users/tokens/edit.html.haml

```diff
@@ -49,12 +49,18 @@
                     Leaving this text field empty will not update your SCM token.
                 .form-group#workflow-configuration-path-group
                   = f.label(:workflow_configuration_path, 'Path for Workflows Configuration File:')
-                  %abbr.text-danger{ title: 'required' } *
                   .input-group
-                    = f.text_field(:workflow_configuration_path, class: 'form-control', required: true,
+                    = f.text_field(:workflow_configuration_path, class: 'form-control',
                                                                  placeholder: 'Eg: .my_subdir/my_workflows_file.yml')
                   .form-text.text-muted
-                    The default path is '.obs/workflows.yml'
+                    The default path is '.obs/workflows.yml'.
+                .form-group#workflow-configuration-url-group
+                  = f.label(:workflow_configuration_url, 'URL to Workflows Configuration File:')
+                  .input-group
+                    = f.text_field(:workflow_configuration_url, class: 'form-control',
+                                                                placeholder: 'Eg: https://example.com/.my_subdir/my_workflows_file.yml')
+                  .form-text.text-muted
+                    When the URL is given, the path will be ignored. It should be a public URL.
```

Comment:
I think we should rephrase this. The URL has to be accessible for the OBS instance, which might be public or not, this we don't know.